src/materials/PhysicalPathTracingMaterial.js

```diff
@@ -215,6 +218,10 @@ export class PhysicalPathTracingMaterial extends MaterialBase {
 
 							}
 
+							if( material.vertexColors ){
+								albedo *= vertexColor;
+							}
```

Comment:
nit: can we add spaces inside the braces and before the first brace to keep in style with the file. Looks like there are some other places in the shaders that need to be updated, as well.